kratos/includes/kratos_application.h

```diff
@@ -31,6 +31,36 @@
 #include "utilities/quaternion.h"
 #include "includes/master_slave_constraint.h"
 #include "includes/linear_master_slave_constraint.h"
+#include "includes/geometrical_object.h"
+
+/* Geometries definition */
+#include "geometries/register_kratos_components_for_geometry.h"
```

Comment:
Remark:
`kratos_application.h` is included in all applications, hence this adds a lot of includes everywhere!!!
I see that this is not related to this PR, but we should refactor this to a different file IMO
@KratosMultiphysics/technical-committee 
If we decide to do this then in a follow-up PR

Reply:
This include is basically geometry.h and node.h, it does not suppose a lot. We just splitted in order to remove the dependency of the node.h from geometry.h

Reply:
I know
I meant those: https://github.com/KratosMultiphysics/Kratos/blob/core/register_quad_geometries/kratos/includes/kratos_application.h#L23-L63

Reply:
OH, yes, I understand, I do not know how to solve that, except with the evil forward declaration (and therefore using pointers in order to compile it)

Reply:
I agree with you @philbucher that this needs a refactoring. Please note that all these members will be the prototype list in the future registry class. So they will be removed from here.